mongomock/aggregate.py

```diff
@@ -358,6 +358,15 @@ def _handle_date_operator(self, operator, values):
             return out_value.second
         if operator == '$millisecond':
             return int(out_value.microsecond / 1000)
+        if operator == '$dateToString':
+            if '%L' in out_value['format']:
```

Comment:
Please also add `NotImplementedError` when using `onNull` or `timezone` parameters.

Reply:
add code to raise OperationFailure